src/test/resources/application.properties

```diff
@@ -1,5 +1,5 @@
-spring.datasource.url=jdbc:mysql://tnbtesting.cjsgdmkb5gwc.us-east-2.rds.amazonaws.com:3306/airtnbSchema
-spring.datasource.username=admin
+spring.datasource.url=jdbc:mysql://localhost:3306/airtnbTesting
```

Comment:
I would double check with Joe/Jessie (I forgot who set up the rds for the testing db) but we might want to keep the rds properties unless they say so.  It may make sense to just have a local database for testing, but I would double check with joe/jessie about this as I don't recall who has ownership of the testing rds.

Reply:
Jesse owns the test database; but if this works I think he would also prefer to not have a live DB for it!

Reply:
So as long as Eliot was able to run his tests I think this is better